references/classification/train.py

```diff
@@ -9,7 +9,7 @@
 import torchvision
 import transforms
 import utils
-from references.classification.sampler import RASampler
+from sampler import RASampler
```

Comment:
Good catch.

Reply:
I will need to patch this urgently as it is interfering with an investigation. See https://github.com/pytorch/vision/pull/4997#discussion_r765129140

Thanks for reporting @xiaohu2015